src/render.js

```diff
@@ -0,0 +1,83 @@
+const renderList = (listToDo) => {
+  const sortedTodo = listToDo.list.sort((a, b) => a.index - b.index);
+  const tasksToDo = document.querySelector('.tasks');
+
+  tasksToDo.innerHTML = '';
+  sortedTodo.forEach((todo) => {
+    const taskContainer = document.createElement('ul');
+    const taskEl = document.createElement('li');
+    const spanEl = document.createElement('span');
+    // const square = document.createElement('i');
+    const removeIcon = document.createElement('i');
+    const remove = document.createElement('button');
+    const textToDo = document.createElement('input');
+    const checkbox = document.createElement('input');
+
+    tasksToDo.appendChild(taskContainer);
+    taskContainer.classList.add('second');
+    taskContainer.appendChild(taskEl);
+    taskEl.appendChild(spanEl);
+    spanEl.classList.add('grey');
+    spanEl.appendChild(checkbox);
+    checkbox.classList.add('right', 'checkbox');
+    checkbox.setAttribute('type', 'checkbox');
+    spanEl.appendChild(textToDo);
+    textToDo.classList.add('edit-text');
+    textToDo.setAttribute('id', todo.index);
+    textToDo.value = todo.description;
+    textToDo.readOnly = true;
+    taskEl.appendChild(remove);
+    remove.classList.add('left', 'grey');
+    remove.appendChild(removeIcon);
+    remove.setAttribute('id', todo.index);
+    remove.style.border = 'none';
+    removeIcon.classList.add('iconR', 'fas', 'fa-ellipsis-v', 'fa-2x');
+  });
+
+  const clear = document.createElement('p');
+  tasksToDo.appendChild(clear);
+  clear.classList.add('grey');
+  clear.innerText = 'Clear all completed';
```

Comment:
- [ ] The clear all completed button doesn't clear the tasks marked as complete, kindly fix this.